R/Task.R

```diff
@@ -79,9 +73,9 @@
 #'   Should sanity of data be checked initially at task creation?
 #'   You should have good reasons to turn this off (one might be speed).
 #'   Default is \code{TRUE}.
-#' @param spatial [\code{logical(1)}]\cr
-#'   Does the task contain a spatial reference (coordinates) which should be used for
-#'   spatial partioning of the data? See details.
+#' @param coordinates [\code{data.frame}]\cr
+#'   X and Y coordinates of a spatial data set that will be used for spatial partitioning of the data in a cross-validation resampling setting.
+#'   Coordinates have to be in "Universal Transverse Mercator" (UTM) format, i.e. numeric values.
```

Comment:
Document the format -- columns named `x` and `y`, same number of rows as data?

Reply:
Updated it in the public part but forgot it in the internal part. Why are there two docs actually?